data/json/furniture_and_terrain/furniture-medical.json

```diff
@@ -573,7 +573,7 @@
     "type": "furniture",
     "id": "f_electron_microscope",
     "name": "electron microscope",
-    "description": "An enormous tool for using electron reflections off a surface to see what very tiny things look like.  Amazing for taking gross pictures of bugs.",
+    "description": "An enormous obersvational tool for studying the details of samples on an immensely small scale.",
```

Comment:
Just a typo: observational.

Reply:
Oh thank _christ,_ I was really hoping somebody who knows what this stuff means haha, can you tell that I flunked chemistry? Anyway, jokes aside, thank you very much, I'll make these revisions.